src/common/models/Projections.ts

```diff
@@ -113,6 +113,14 @@ export class Projections {
     }
   }
 
+  getMetricValues(): { [metric in Metric]: number | null } {
```

Comment:
This code was duplicated in several places (and would have required updating to use `projection.icuHeadroomInfo.metricValue` now), so I consolidated it here.